### PR TITLE
Updating share buttons with their respective brand colors

### PR DESCRIPTION
**Pull request by walmyrcarvalho**
:rocket:

### CODE REVIEW

File: css/share.css [hunk 1]
```diff
@@ -6,4 +6,20 @@ ul.share {
...
+    color: #3B5998;
+}
+
+.fa-linkedin-square:hover {
+    color: #007BB5;
+}
+
+.fa-google-plus-square:hover {
+    color: #DD4B39;
 }
```
Comment (walmyrcarvalho) [inline css/share.css:25]: Ops, sorry. :(

Comment (marcelofabri) [inline css/share.css:25]: não entendi :sweat_smile:

Comment (walmyrcarvalho) [inline css/share.css:25]: @marcelofabri ele deu aquele alerta de missing line no final. :v

Comment (marcelofabri) [inline css/share.css:25]: ah, acho q já tava assim ¯_(ツ)_/¯

**Comment by marcelofabri** [comment]
:ship:

**Pull request merged**